requirements.yml

```diff
@@ -39,8 +39,8 @@
   version: 0.1.0
 
 - name: openmicroscopy.omero-prometheus-exporter
```

Comment:
This can now be updated as 

```
- src: openmicroscopy.omero_prometheus_exporter
   version: 0.2.0
```

which installs the latest `omero-promethus-tools` package.